github/prci.py

```diff
@@ -117,6 +117,9 @@ def process_pull_request(
                 logger.warning(e)
 
     for name, task_data in tasks_data.items():
+        if "job" not in task_data:
```

Comment:
I would rather modify the `__init__` method of the class Task and put the check in `__init__`, just before the line `job_data = task_data["job"]`
When the check fails, return an Exception and use try/except around `task = Task(...).`
This looks better to me as the `__init__` method should be responsible for its own args validation.

Reply:
updated...